src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArr = people.filter(
+    person => person.sex === 'm'
+      && (century === undefined
```

Comment:
```suggestion
      && (!century 
```
використовуй скорочений запис